packages/material/src/complex/MaterialTableControl.tsx

```diff
@@ -96,6 +96,7 @@ const generateCells = (
       const props = {
         propName: prop,
         schema,
+        title: schema.properties?.[prop]?.title ?? startCase(prop),
```

Comment:
now that this conditional is just one line (thank you for teaching me new syntax!) I don't think it needs to be a variable anymore